content/en/tracing/guide/monitor-kafka-queues.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Tracing Kafka queues
+kind: guide
+further_reading:
+- link: "/tracing/trace_collection"
+  tags: "Documentation"
+  text: "Set up trace collection"
+- link: "/integrations/kafka"
+  tags: "Documentation"
+  text: "Kafka integration"
+- link: "/tracing/troubleshooting/"
+  tags: "Documentation"
+  text: "Data Streams Monitoring"
+---
+
+## Monitor Kafka queues
+
+In event-driven pipelines, queuing and streaming technologies such as Kafka are essential to the successful operation of your systems. However, ensuring that messages are being reliably and quickly conveyed between services is difficult due to the many technologies and teams involved in any such environment. Datadog's Kafka integration and its APM suite enables your team to monitor your infrastructure and pipelines health and efficiency.
+
+### Integration
+
+Easily visualize the performance of your cluster in real time qnd correlate the performance of Kafka with the rest of your applications by using [Datadog's Kafka integration][1]
```

Comment:
type: `qnd` --> `and`

Reply:
Note: You could also mention the MSK integration if customers use MSK

Reply:
Thanks . Fixed in d674520d612c9a53638d16682090771de6c96843